src/GitVersion.Configuration/ConfigurationFileLocator.cs

```diff
@@ -13,6 +13,9 @@ internal class ConfigurationFileLocator(
 {
     public const string DefaultFileName = "GitVersion.yml";
     public const string DefaultAlternativeFileName = "GitVersion.yaml";
+    public const string DefaultFileNameDotted = $".{DefaultFileName}";
+    public const string DefaultAlternativeFileNameDotted = $".{DefaultAlternativeFileName}";
+    public List<string> PossibleConfigFileNames = [DefaultFileName, DefaultAlternativeFileName, DefaultFileNameDotted, DefaultAlternativeFileNameDotted];
```

Comment:
Perhaps `Supported` is a better word to use here? 

```suggestion
    public List<string> SupportedConfigFileNames = [DefaultFileName, DefaultAlternativeFileName, DefaultFileNameDotted, DefaultAlternativeFileNameDotted];
```

Reply:
Agree with the suggestion